README.md

```diff
@@ -122,6 +123,10 @@ Configure if HAProxy should maintain client requests to the same backend server.
 * `ingress.kubernetes.io/affinity`: the only supported option is `cookie`. If declared, clients will receive a cookie with a hash of the server it should be fidelized to.
 * `ingress.kubernetes.io/session-cookie-name`: the name of the cookie. `INGRESSCOOKIE` is the default value if not declared.
 * `ingress.kubernetes.io/session-cookie-strategy`: the cookie strategy to use (insert, rewrite, prefix). `insert` is the default value if not declared.
+* `ingress.kubernetes.io/session-cookie-dynamic`: indicates whether or not dynamic cookie naming will be used. With the
+default of `true`, a cookie will be generated based on the ip address and port of the server, along with any specified
```

Comment:
... a cookie will be generated as a hash of the ip address and port of ... (something like this)

Reply:
Fixed.